### PR TITLE
Add ingress-nginx chart

### PR DESCRIPTION
**Pull request by aledbf**
cc @ChiefAlexander

### CODE REVIEW

**Comment by rimusz** [comment]
@aledbf thanks a lot for the PR, would you mind to add an email as well, this is how we validate the user, thanks a lot

File: repos.yaml [hunk 1]
```diff
@@ -10,3 +10,10 @@
 #      - name: foo
 #        email: foo@example.com
 #
+
+repositories:
+  - name: kubernetes/ingress-nginx
```
Comment (elioengcomp) [inline repos.yaml:15]: Namespaces cannot contain slashes

Comment (rimusz) [reply repos.yaml:15]: it should look as `kubernetes-ingress-nginx`

Comment (aledbf) [reply repos.yaml]: fixed

**Pull request merged**